LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentRequest.java

```diff
@@ -206,6 +222,10 @@ public String getSurname() {
         return surname;
     }
 
+    public boolean hasUserLocationConsent() {
```

Comment:
I think we will want to docstring this as well

Reply:
I saw that the above getter functions do not have any docstrings. I feel that merchants would not realistically use these getters, since they'd be the ones to instantiate these request objects. We could probably mark them as internal, but maybe a larger discussion outside the scope of this PR.

Would we still want to add a docstring here?

Reply:
🤔 interesting, it looks like we are pretty inconsistent across the SDK, for example [3DS has them docstringed](https://github.com/braintree/braintree_android/blob/6f2324098431753ac74778edaa28d2f7cf4d1b1c/ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java#L389) but other requests don't. I'm curious what other folks think or if this is tech debt that we should consider addressing in v5 (either docstring them all or making them internal).

Reply:
I think tackling this in v5 makes sense but also open to what other folks think

Reply:
+1 to adding this to the v5 backlog

Reply:
I _think_ the getters need to exist for Kotlin interop (making the property accessible), but not 100% sure on that, and if we convert these request objects to Kotlin in v5 might be resolved on it's own

Reply:
Yep, I thought about having the getter named `isHasUserLocationConsent()` which would make it property accessible in Kotlin, but the only use of this property is in Java files. I think you're right in that when we do the conversion to Kotlin, the issue will be resolved.

Reply:
Oh but don't we want merchants to be able to set this property if they are integrating in Kotlin?

Reply:
Since it's a required field, I was thinking it'd be defined as a `val` in the constructor when we convert to a Kotlin data class. So it wouldn't have a separate setter function generated.

I may have prematurely made this assumption 😅 Is this the direction we'd want to go in?